README.md

```diff
@@ -1,9 +1,13 @@
 # beaker-http
 
-This library is designed to assist in test scenarios where http requests to a Beaker::Host
-are required. It utilizes the Faraday library to generate requests, and the [Http](lib/beaker-http/http.rb)
-class is designed to be subclassed into objects that are more targeted to specific http
-services running on beaker hosts.
+This library adds the ability to send http traffic from the beaker coordinator itself,
+reducing the need to use beaker's DSL method `on` as an interface to `curl` on a SUT.
+It utilizes the Faraday library to generate requests, and the [Connection](lib/beaker-http/http.rb)
+class in the `Beaker::Http` module is the class you want to use directly, either utilizing it
+directly or subclassing it to build your own Connection class.
+
+To use the class directly, please use the DSL methods included in this library [here](lib/beaker-http/dsl/web_helpers.rb). Use the `generate_new_http_connection` to reuse an object multiple times
+throughout a test, or use the `http_request` method to send single requests.
```

Comment:
why don't we publish rubydocs about this, and link to there rather than the code? That will explain things a little easier to the reader, and you can always read code from there if needed.